packages/docs/.storybook/preview.ts

```diff
@@ -92,4 +189,4 @@ const preview: Preview = {
   },
 };
 
-export default preview;
+export default preview;;
```

Comment:
A second `;` has sneaked in

```suggestion
export default preview;
```